Runtime/AgentDispatcher.cs

```diff
@@ -48,22 +48,21 @@ public AgentDispatcher(AutopilotSettings settings, ILogger logger, RandomFactory
             _settings = settings;
             _logger = logger;
             _randomFactory = randomFactory;
+            SceneManager.sceneLoaded += this.DispatchByScene;
         }
 
-        /// <summary>
-        /// Dispatch agent mapped to Scene `next`
-        /// </summary>
-        /// <param name="current"></param>
-        /// <param name="next"></param>
-        public void DispatchByScene(Scene current, Scene next)
+        public void Dispose()
+        {
+            SceneManager.sceneLoaded -= this.DispatchByScene;
+        }
+
+        /// <inheritdoc/>
+        public void DispatchByScene(Scene next, LoadSceneMode mode)
```

Comment:
I think it is better to make private the method. Because it is used in only internals, and the 2nd argument is meaningless.

Reply:
This method is defined in "IAgentDispatcher", so it cannot change to lower accessor.
Solution: Can I remove "IAgentDispatcher"? it is not effective as an interface.

Reply:
Did you mean that "remove IAgentDispatcher interface?" Or, did you mean "remove `DispatchByScene` from IAgentDispatcher?"

Reply:
If previous one is true, I won't agree it, because `DispatchByScene(Scene)` exist. Otherwise, I'll agree it.

Reply:
Thank you. I meant the previous one.
But you're right. I remove `DispatchByScene(Scene, LoadSceneMode)` method from 'IAgentDispatcher'.

Reply:
Fixed in faf185c